Elevel/Application/Infrastructure/ValidationException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Infrastructure
+{
+    class ValidationException: Exception
+    {
+        public ValidationException(string message) 
+            :base(message) 
+        {
+
+        }
```

Comment:
Best practice is use three overloads
https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions